src/commands/step-functions/README.md

```diff
@@ -0,0 +1,67 @@
+# step-functions commands
+
+You can use the `step-functions instrument` command to instrument your Step Functions with Datadog. This command enables instrumentation by subscribing step function logs to a [Datadog Forwarder](https://docs.datadoghq.com/logs/guide/forwarder/).
+
+You can also add the `step-functions instrument` command to your CI/CD pipelines to enable Datadog instrumentation for all of your step functions. Run the command after your normal serverless application deployment, so that changes made by this command do not get overridden by changes in the CI/CD pipeline.
+
+## Usage
+
+### instrument
+
+Run the `instrument` command to subscribe a Step Function log group to the specified Datadog Forwarder. If either `logLevel: ALL` or `includeExecutionData: true` is not set, then the Step Function logging configuration will be updated to use those settings.
```

Comment:
I don't understand where `logLevel: ALL` or `includeExecutionData: true` come from. Are these environment variables from the Datadog Forwarder, or the Step Function? Or are these settings from AWS? I'm confused

Reply:
These 2 configs come from the requirements of Step Functions tracing instrumentation google doc, which I've just shared with you @duncanista.

Reply:
Thanks!